source/guides/clear/stateless.rst

```diff
@@ -65,14 +65,14 @@ Default configurations
 Software in |CL| provides default configuration values so that it is
 immediately functional, whenever it is appropriate to do so.
 
-|CL| distributed software packages may be directly modified to include default
+|CL| distributed software may be directly modified to include default
 configuration values or default configuration files may be provided by |CL|
 under :file:`/usr/share/defaults`. These files can be referenced as templates
 for customization.
 
-For example, the default configuration that Apache uses when installed can be
-found at :file:`/usr/share/defaults/httpd/httpd.conf` directory.
-
+As an example, default configurations that the Apache web server uses,
+after installing the `httpd` bundle, can be found in the 
+:file:`/usr/share/defaults/httpd/` directory.
```

Comment:
For example, if you want to use the Apache web server, install the `httpd` bundle, as shown in the next section.  

[I think it's best to keep this simple. No need to explain what you're already showing how to do in next section.]

Reply:
I see them as 2 separate topics.  The first is "defaults".  The 2nd is how to override.  
it just so happens that we're using Apache as the example in both topics.

Reply:
OK. I can live with that. I see your point.  How about this: 

For example, after installing the `httpd` bundle for Apache web server, default configurations appear in :file:`/usr/share/defaults/httpd/` directory.


I found it hard to understand the writer's intent.

Reply:
done.

Reply:
Thank you.